clients/python/test/e2e/conftest.py

```diff
@@ -71,3 +72,28 @@ def sleeper(api_client: osparc.ApiClient) -> osparc.Solver:
         "simcore/services/comp/itis/sleeper", "2.0.2"
     )  # type: ignore
     return sleeper
+
+
+@pytest.fixture
+def sleeper_study_id(api_client: osparc.ApiClient) -> UUID:
+    """Simple sleeper study template which takes
+    as input a single file containing a single integer"""
+    _test_study_title = "sleeper_test_study"
```

Comment:
Is this study expected in the deploy? 
Imaging it fails ... What do you do? If so explain how is it created etc  Can use some  links to instructions

Reply:
Yes, this is expected in the deploy. I raise an Assertion error if the study cannot be find. So that case is handled. I believe the doc explains how the study is set up.

Reply:
I meant "how to setup from scratch this study in case it is deleted/broken" ?:-)